App/CrisisCleanup/Sources/Feature/Cases/CasesViewModel.swift

```diff
@@ -2,11 +2,24 @@ import SwiftUI
 import Combine
```

Comment:
There's copy-pasting of a lot of unnecessary code. Remove anything not used. It's only going to clutter these classes that are going to get much more complex once more features go in.